pandas/tests/series/test_constructors.py

```diff
@@ -52,6 +52,16 @@
 
 
 class TestSeriesConstructors:
+    def test_unparseable_strings_with_dt64_dtype(self):
```

Comment:
Worth adding a similar test for DataFrame construction? Or is the path shared?

Reply:
the edited test_block_internals test hits DataFrame